Makefile

```
@@ -83,3 +86,10 @@ clean:
 			done
 		fi
 	done
+
+# Generate environment
+.PHONY: env
+env: 
+	@export $(find . -name '*.env' -exec grep -rhv -e '^#' -e '^$' {} + | xargs)
+	@export MORPH_BLOCK_TIME=$(grep 'SecondsPerBlock' $(MORPH_CHAIN_PROTOCOL) | tail -n 1 | cut -d: -f2) 
```

1. Because it is Makefile and not shell, you need 
```
MORPH_BLOCK_TIME=$(shell grep 'SecondsPerBlock...
```

2. I don't think valid protocol file will have more than one `SecondPerBlock` line. So we can drop `tail -n 1` part.
3. Cutting by `:` keeps whitespace before number
```
MORPH_BLOCK_TIME= 1
```
I would use `awk` instead of `cut`. And in the end it would look like this:
```suggestion
	@export MORPH_BLOCK_TIME=$(shell grep 'SecondsPerBlock' $(MORPH_CHAIN_PROTOCOL) | awk '{print $$2}')
```